pkg/utils/containerId.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+func ContainerId(containerIdBefore [9]string, podNs string, podLabels string, client *kubernetes.Clientset) bool {
```

Comment:
pls use some meaningful names i.e, checkContainerID

Reply:
You are using so much of arguments for this function, can you please try to reduce the no. of arg ( try structure)

Reply:
These are the names of tests  i.e `containerId` test `RestartCount` test `ResourceVersion` test. `checkContainerId` seems odd for test name isn't it?

Reply:
BCH?